rust/src/quic/quic.rs

```diff
@@ -327,11 +360,21 @@ impl QuicState {
                             None,
                             None,
                             to_server,
+                            false,
                         );
                         continue;
                     }
 
-                    match QuicData::from_bytes(framebuf) {
+                    let mut frag = Vec::new();
+                    // take the current fragment and reset it in the state
```

Comment:
I don't understand what is happening here, can you explain?

Reply:
In C it would be like :
```
uint8_t * frag = state->frag_ts;
state->frag_ts = NULL;
```